src/GitVersionHelper.php

```diff
@@ -63,4 +63,36 @@ public static function getNameAndVersion()
     {
         return self::appName() . '/' . self::getVersion();
     }
+    
+    /**
+     * Returns comment hash with custom length
+     *
+     * @param null $length
+     * @return string
+     *
+     */
+    public static function getCommitHash($length=null)
+    {
+        // Remember current directory
+        $dir = getcwd();
+
+        // Change to base directory
+        chdir(base_path());
+
+        // Get version string from git
+        $commit = shell_exec('git rev-parse HEAD');
+
+        // Change back
+        chdir($dir);
+
+        if ($commit === null) {
+            throw new Exception\CouldNotGetVersionException;
+        }
+
+        if($length) {
```

Comment:
Follow PSR-2 code style. You can use [php-codesniffer(https://github.com/squizlabs/PHP_CodeSniffer) such as `php phpcs.phar --standard=PSR2 *.php` to look for issues.